packages/flame/lib/src/game/camera/camera_wrapper.dart

```diff
@@ -1,34 +1,38 @@
 import 'dart:ui';
 
+import 'package:meta/meta.dart';
+
 import '../../../components.dart';
 import 'camera.dart';
 
 /// This class encapsulates FlameGame's rendering functionality. It will be
 /// converted into a proper Component in a future release, but until then
 /// using it in any code other than the FlameGame class is unsafe and
 /// not recommended.
-class CameraWrapper {
-  // TODO(st-pasha): extend from Component
+@internal
+class CameraWrapper extends Component {
```

Comment:
```suggestion
class _CameraWrapper extends Component {
```

Reply:
If it is private we have to define it inside of `flame_game.dart` right? `@internal` and not being exported should be enough?